main.tf

```diff
@@ -1,4 +1,6 @@
-variable "aws_region" { default = "eu-west-2" } # London
+# Ireland - doesn't yet seem to be available in London. See https://aws.amazon.com/marketplace/pp/B06VSPXKDX?qid=1504282131156&sr=0-1&ref_=srh_res_product_title
+
+variable "aws_region" { default = "eu-west-1" }
```

Comment:
We can currently pass the region as parameter, I wouldn't change the default one just to be able to use the GPU image

Reply:
yep I agree -- these are old commits before we made a lot of changes, just created a pull request to make them visible!